TowerNinja/Assets/Scripts/friendDmgHitbox.cs

```diff
@@ -36,6 +40,12 @@ void Update()
         {
             damageTimer = 0;
         }
+
+        if (totalCollisions == 0)
+        {
+            damageBool = false;
+            this.GetComponent<Rigidbody2D>().velocity = velocity;
```

Comment:
we could probably save `this.GetComponent<Rigidbody2D>()` as a field in the behavior